DuckDuckGo/BookmarksViewController.swift

```diff
@@ -50,6 +50,17 @@ class BookmarksViewController: UITableViewController {
             selectLink(link)
         }
     }
+    
+    @available(iOS 11.0, *)
+    override func tableView(_ tableView: UITableView, leadingSwipeActionsConfigurationForRowAt indexPath: IndexPath) -> UISwipeActionsConfiguration? {
+        let shareContextualAction = UIContextualAction.init(style: .normal, title: "Share") { (_, _, completionHandler) in
+            Logger.log(text: "share action")
```

Comment:
I believe we can remove that and only log error cases, rather than regular usage.